GitUI/Editor/ContinuousScrollEventManager.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using GitCommands;
+
+namespace GitUI.Editor
+{
+    public class ContinuousScrollEventManager
```

Comment:
```suggestion
    internal sealed class ContinuousScrollEventManager
```

Reply:
Must be public, FileViewerInternal.cs SetContinuousScrollManager() uses the class (that method could be set internal too)

Reply:
No worries